libflux/flux-core/src/semantic/types.rs

```diff
@@ -13,20 +13,27 @@ use std::{
 
 /// For use in generics where the specific type of map is not mentioned.
 pub type SemanticMap<K, V> = BTreeMap<K, V>;
+#[allow(missing_docs)]
 pub type SemanticMapIter<'a, K, V> = std::collections::btree_map::Iter<'a, K, V>;
 
+/// A polytype.
```

Comment:
PolyType is a type scheme that quantifies the free variables of a monotype. 

vars- The list of vars are the free variables within the monotypes
cons - The list of kind constraints on any of the free variables

expr - the underlying monotype